packages/react/src/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PropTypes from 'prop-types';
+import React, { AriaAttributes, PropsWithChildren } from 'react';
+import cx from 'classnames';
+import Link from '../Link';
+import { OverflowMenuHorizontal } from '@carbon/icons-react';
+import { usePrefix } from '../../internal/usePrefix';
+import { ForwardRefReturn } from '../../types/common';
+import { Text } from '../Text';
+
+export interface BreadcrumbItemProps
+  extends React.HTMLAttributes<HTMLLIElement> {
+  'aria-current'?: AriaAttributes['aria-current'];
```

Comment:
The TypeScript types aren't bound by semver right now and are provided on an as-is basis. I think it's fine to specify this here. It won't be classified as a breaking change.